test_files/declare.sickle.ts

```diff
@@ -1,5 +1,17 @@
-declare module DeclareTest {
+declare module DeclareTestModule {
   export class Foo {
     constructor();
   }
 }
+
+declare interface DeclareTestInterface {
```

Comment:
It also works to use an @interface here, but I was afraid of that because I don't know the full consequences of all these crazy Closure attributes.  (Like I think you meant @struct, not @record ?)